src/importexport/musicxml/internal/musicxml/importmxmlnotepitch.cpp

```diff
@@ -143,6 +143,11 @@ void MxmlNotePitch::pitch(muse::XmlStreamReader& e)
                 if (ok2 && (std::abs(altervalue) < 2.0) && (m_accType == AccidentalType::NONE)) {
                     // try to see if a microtonal accidental is needed
                     m_accType = microtonalGuess(altervalue);
+
+                    // If it's not a microtonal accidental we will use tuning
+                    if (m_accType == AccidentalType::NONE) {
+                        m_tuning = 100 * altervalue;
```

Comment:
I'd initialize it (to `0,0')  further abone, where `alter` etc. are initialized too

Reply:
Done